rmw_zenoh_cpp/src/detail/rmw_client_data.hpp

```diff
@@ -64,7 +65,7 @@ class ClientData final : public std::enable_shared_from_this<ClientData>
   bool liveliness_is_valid() const;
 
   // Copy the GID of this ClientData into an rmw_gid_t.
-  void copy_gid(uint8_t out_gid[RMW_GID_STORAGE_SIZE]) const;
+  std::vector<uint8_t> copy_gid() const;
```

Comment:
My opinion here is that we should actually go with this new signature; while I don't love that the copy is somewhat implicit, that is just C++ rules.  I'm mostly fine with this because the method has the word "copy" in its name.

Reply:
That said, we are noticing additional latency with this port, so we should check to see if this is the cause of it.

Reply:
I think with #340 merged, this is resolved, so resolving this comment.